showcase/src/app/demo-ui/pages/date-range-picker/demo-date-range-picker-page.component.ts

```diff
@@ -1,30 +1,66 @@
-import { Component, Inject } from "@angular/core";
+/* tslint:disable:no-null-keyword trackBy-function */
+import { Component, Inject, OnDestroy } from "@angular/core";
 import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
-import { StarkDatePickerFilter, StarkDateRangePickerEvent, StarkTimestampMaskConfig } from "@nationalbankbelgium/stark-ui";
+import { AbstractControl, FormControl, FormGroup, ValidationErrors, Validators } from "@angular/forms";
+import { MatCheckboxChange } from "@angular/material/checkbox";
+import { Subscription } from "rxjs";
 import { ReferenceLink } from "../../../shared/components";
+import { StarkDateRangePickerEvent } from "@nationalbankbelgium/stark-ui";
+import map from "lodash-es/map";
 
-const DAY_IN_MILLISECONDS = 86400000;
+const MONTH_IN_MILLI = 2592000000;
 
 @Component({
 	selector: "demo-date-range-picker",
 	templateUrl: "./demo-date-range-picker-page.component.html"
 })
-export class DemoDateRangePickerPageComponent {
-	public startDate?: Date;
-	public endDate?: Date;
+export class DemoDateRangePickerPageComponent implements OnDestroy {
+	public static noFebruaryValidator(control: AbstractControl): ValidationErrors | null {
+		const { value } = control;
+		return value instanceof Date && value.getMonth() === 1 ? { inFebruary: true } : null; // date counts months from 0
+	}
+
+	public today = new Date();
+	public inOneMonth = new Date(this.today.getTime() + MONTH_IN_MILLI);
 
-	public minDate = new Date();
-	public maxDate = new Date(Date.now() + 30 * DAY_IN_MILLISECONDS);
+	public dateRangeModel = { startDate: this.today, endDate: this.inOneMonth };
+	public modelDisabled = false;
 
-	public customDateFilter: StarkDatePickerFilter = (date: Date): boolean => date.getDay() !== 0;
+	public dateRangeFormGroup = new FormGroup({
+		startDate: new FormControl(null, Validators.compose([DemoDateRangePickerPageComponent.noFebruaryValidator, Validators.required])),
+		endDate: new FormControl(null, Validators.compose([DemoDateRangePickerPageComponent.noFebruaryValidator, Validators.required]))
+	});
 
-	public customDateMask: StarkTimestampMaskConfig = {
-		format: "DD-MM-YYYY"
-	};
+	public getErrorMessages(control: AbstractControl): string[] {
+		return map(
```

Comment:
Question:  what is the added value of using Lodash here instead of simply Array.map() ?

Reply:
control.errors is an object, so I'm not able to use the Array.map(). But I should change the fallback 😅